.github/workflows/sync-version-with-upstream.yml

```diff
@@ -25,16 +25,19 @@ jobs:
           )
           sed -i 's/^\(version: \).*$/\1'"$VERSION"'/' snap/snapcraft.yaml
 
-          # Fetch the upstream package.json for the fetched version
-          wget -qO package.json https://raw.githubusercontent.com/signalapp/Signal-Desktop/v${VERSION}/package.json
```

Comment:
Honestly, I still don't understand why this approach didn't work. Does this mean signal updated their `package.json` for an _already released_ signal version?

Reply:
Or could it be that our list of starting versions was simply incorrect and the automatic hook simply corrected it?

Reply:
Difficult to say. The [upstream 6.38.0](https://github.com/signalapp/Signal-Desktop/blob/eee8ee37073a8d4e89a44d8fde75e788e2518cd1/package.json#L100C27-L100C36) does indeed use `2.34.0`, so I wonder if there was just a mistake to start with :man_shrugging: